src/providers/pubsub.ts

```diff
@@ -137,6 +142,14 @@ export class ScheduleBuilder {
     return this;
   }
 
+  /**
+   * Event handler that fires when a scheduled event occurs for
```

Comment:
Language is a bit weird here - maybe something like "Event handler for scheduled functions. Triggered whenever the associated scheduler job sends a PubSub message"?

Reply:
+1